core/src/main/java/org/infinispan/cache/impl/CacheImpl.java

```diff
@@ -129,6 +129,8 @@
 @MBean(objectName = CacheImpl.OBJECT_NAME, description = "Component that represents an individual cache instance.")
 public class CacheImpl<K, V> implements AdvancedCache<K, V> {
    public static final String OBJECT_NAME = "Cache";
+   private static final EnumSet<Flag> IGNORE_RETURN_VALUES_FLAG_SET = EnumSet.of(IGNORE_RETURN_VALUES);
```

Comment:
btw, is it worth it? I mean, we safe one allocation but every time you add or remove flags you have to make a copy...